packages/sync/src/class-defaults.php

```diff
@@ -1176,4 +1182,67 @@ public static function is_multi_network() {
 	 * @var int 1 for true.
 	 */
 	public static $default_full_sync_sender_enabled = 1; // Should send full sync items.
+
+	/**
+	 * Default Full Sync config
+	 *
+	 * @var array list of module names.
+	 */
+	public static $default_full_sync_config = array(
+		'constants'          => 1,
+		'functions'          => 1,
+		'options'            => 1,
+		'updates'            => 1,
+		'themes'             => 1,
+		'users'              => 1,
+		'terms'              => 1,
+		'posts'              => 1,
+		'comments'           => 1,
+		'term_relationships' => 1,
+	);
+
+	/**
+	 * Send all Items
```

Comment:
This description needs some ice cream and love.

Reply:
Finally I renamed it to `` https://github.com/Automattic/jetpack/pull/13963/commits/69933ced3c8e1e14023c91067fde47e019f86f6b